objects.go

```diff
@@ -2,12 +2,15 @@ package main
 
 import "github.com/faiface/pixel"
 
-// Gravity is vector representing standard gravity force accelleration vector
+// Gravity is vector representing standard gravity force accelleration vector in m*s^{-2}
 var Gravity = pixel.Vec{
 	X: 0.0,
 	Y: -9.81,
 }
 
+// PixelsPerMeter is the number of pixels on screen that represent one meter in real life
+var PixelsPerMeter = 100.0
```

Comment:
Are we going to change this variable during the runtime? If no, `const` might be better.